go/libraries/doltcore/doltdb/root_val.go

```diff
@@ -558,6 +557,43 @@ func (root *RootValue) HasConflicts(ctx context.Context) (bool, error) {
 	return len(cnfTbls) > 0, nil
 }
 
+func (root *RootValue) IterTables(ctx context.Context, cb func(name string, table *Table) (stop bool, err error)) error {
```

Comment:
profileration of "stop" instead of "continue" return variable.  Should figure out if we actually want to fixk this.